packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.2
```

Comment:
These should all be minor version bumps, not bugfix version bumps, as it's adding a new feature.